src/queryHistory/queryHistoryProvider.ts

```diff
@@ -176,6 +176,7 @@ export class QueryHistoryProvider implements vscode.TreeDataProvider<any> {
                 uri,
                 undefined,
                 title,
+                {},
```

Comment:
Can you add a comment saying which parameter this is meant to be, since it isn't otherwise apparent?